frontend/src/home/Stories.vue

```diff
@@ -1,8 +1,28 @@
 <script setup lang="ts">
 import StoryCard from '@/components/StoryCard.vue'
 
-import { ref } from 'vue'
+import { ref, onMounted } from 'vue'
 import { useRouter } from 'vue-router';
+import Papa from 'papaparse';
+const sid = import.meta.env.VITE_WOMEN_CONTENT_SHEET_ID
+
+const StoriesURI = `https://docs.google.com/spreadsheets/d/${sid}/export?format=csv`
+const rows = ref([]);
+
+async function getData() {
+  const res = await Papa.parse(StoriesURI, {
```

Comment:
Nit: I think `res` is an unused variable. We can just take out the variable assignment.

Same in StoriesView.vue.

Reply:
Fixed, thanks for catching.